evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -367,6 +383,26 @@ contract Operator is
     assert(linkToken.transfer(msg.sender, payment));
   }
 
+  // PUBLIC FUNCTIONS
+
+  /**
+   * @notice Create a new forwarder contract
+   * @dev This function uses create2 to deploy at a predetermined address.
+   * @param salt bytes32 salt for using create2
+   * @return addr deployed address
+   */
+  function createForwarder(bytes32 salt) public onlyOwner() returns (address addr) {
+    require(s_authorizedSenderList.length != 0, "Must have authorized senders");
+    OperatorForwarder forwarder = new OperatorForwarder{salt: salt}();
```

Comment:
This line causes `replaceMetadata` in [extract_contract_details.go](https://github.com/smartcontractkit/chainlink/blob/381b89bdf24ee891979c2723321a977babd97cc4/core/internal/gethwrappers/extract_contract_details.go) to fail by decoding non-ascii data. 

It fails regardless of whether it uses a normal contract creation or `create2` as seen here.